### ISSUE TITLE
License Activation Issue: `environment:null` in Self-Hosted Keygen Running on Docker

### ISSUE THREAD

**Issue by christomecki** [keygen-sh/keygen-go#10]
I'm attempting to activate a license using this package, as illustrated in the example in the [readme#license-activation](https://github.com/keygen-sh/keygen-go#license-activation) but I'm encountering issues with the `self-hosted` Keygen, which I'm running locally on Docker.

The `keygen.Validate(fingerprint)` returns error and then the whole application exits with `panic("license is invalid!")`.

In the Golang `main` function, I've set:
```golang
keygen.Account = "<accountID>"
keygen.Product = "<productID>"
keygen.LicenseKey = "<licenseKey>"
keygen.APIURL = "https://localhost"
```
What's important, I guess, I didn't set `Environment`.

After some debugging, I discovered that for the request:
`https://localhost/v1/licenses/22f4f94f-7236-47eb-8adf-ff85c5ec061c/actions/validate`
with body:
```json
{
   "meta":{
      "scope":{
         "fingerprints":[
            "8f33e2dfbd5cb464ed34fb4afd48a2fc7da402386ef0ced0cb77d487e07ebb61"
         ],
         "product":"2cb576aa-a174-400f-9a4d-c5b9cf1d0b8a",
         "environment":null
      }
   }
}
```

response is:
```json
{
   "meta":{
      "id":"785f922f-cf82-46a6-8a42-619830e9e104"
   },
   "errors":[
      {
         "title":"Bad request",
         "detail":"unpermitted parameter",
         "source":{
            "pointer":"/meta/scope/environment"
         }
      }
   ]
}
```

I suspect the issue arises because the package sends `"environment":null`, and for some reason, this is not accepted.
I managed to fix this modifying package source code like this:
```golang
//https://github.com/keygen-sh/keygen-go/blob/master/validate.go

type scope struct {
	Fingerprints []string `json:"fingerprints"`
	Product      string   `json:"product"`
	Environment  *string  `json:"environment,omitempty"` //added omitempty
}
```

After that it began working, providing the output `"License is activated!"`.

So now I'm trying to figure out what I'm doing wrong - bad self-hosted Keygen configuration, wrong policy/license parameters, a problem with my code, or a problem within the package. 

Thanks for any help you can provide.

**Comment by ezekg** [comment]
Sorry about that. We need to set up a way to test against CE as well as EE.

Fixed in dcfd154bb431561c0ff01be604bf0f9539e9cd9a. Please try v2.6.1.

**Comment by christomecki** [comment]
Now it works, thanks👍
